client/src/constants/csvReports.js

```diff
@@ -0,0 +1,18 @@
+import { saveAs } from 'file-saver';
+import { ToastStatus, DOWNLOAD_DEFAULT_ERROR_MESSAGE, DOWNLOAD_DEFAULT_SUCCESS_MESSAGE } from './';
+
+export const onReportDownloadResult = async (response, reportFileName, openToast) => {
```

Comment:
Agreed! I wonder if `constants` is the correct folder though, I think something like `util` or `services` might make more sense

Reply:
Oooooh yeah I didn't notice that. Between those I like utils best (it already has pdf.js).

Reply:
Utils please!

Reply:
I'm not 100% convinced about passing in the openToast here, is it not possible to use the hook? On the other hand, it may not be a great idea for a "util" to do that.

Reply:
I'll move it to `utils` and see what can be done about `openToast` 👍